locales/sv-SE/src/structure.ts

```diff
@@ -84,6 +84,13 @@ export default removeUndefinedLocaleResources({
   'banners.deleted-document-banner.text': 'Detta dokument har raderats.',
   /** The text content for the deprecated document type banner */
   'banners.deprecated-document-type-banner.text': 'Denna dokumenttyp är inte längre tillgänglig.',
+  /** The text for publish action for discarding the version */
+  'banners.live-edit-draft-banner.discard.tooltip': 'Kassera utkast',
+  /** The text for publish action for the draft banner */
+  'banners.live-edit-draft-banner.publish.tooltip': 'Publicera för att fortsätta redigera',
+  /** The text content for the live edit document when it's a draft */
+  'banners.live-edit-draft-banner.text':
+    'Typen <strong>{{schemaType}}</strong> har <code>liveEdit</code> aktiverat, men en utkastversion av detta dokument finns. Publicera eller kassera utkastet för att fortsätta redigera det live.',
```

Comment:
Should it say `liveEdit` here? Sure, that's what the property is called in the schema, but that might not be very clear to a Swedish-speaking, non-developer Editor. Should it instead say "har live-redigering aktiverat"? Not sure.

Reply:
Note that this occurs in a rare situation where a document type is marked as having `liveEdit` enabled (eg changes are published immediately on any change, there is explicit "publish" action) but there is an existing draft present, or a draft was created through the API or similar.

Reply:
So you're saying this is a message that mostly developers will see and need to deal with? Should be fine in that case! 😊